### DISCUSSION TITLE
Adding Individual Emails

### DISCUSSION THREAD

**Discussion by FetchEOD** [gingerbeardman/stapler#35] [Q&A] [answered]
I would love to be able to add individual emails to my Stapler (v1.2.0) document by dragging and dropping, but this doesn't seem to be possible from either Mail or MailMate in MacOS Ventura 13.4.  

At the moment I am either printing to PDF or copy/pasting the text into a text file.

Thanks for such a useful app.

Andy

**Comment by FetchEOD** [comment]
Updated to Stapler v1.2.4 - the result is the same.

Thanks again.

**Comment by gingerbeardman** [comment, marked as answer]
Great suggestion, I will turn this into an issue for feature request.

There's a PR to add support for webloc which I think would do most of the work required for email support.

You can watch that issue #36 and we can continue discussion there as support is added.
